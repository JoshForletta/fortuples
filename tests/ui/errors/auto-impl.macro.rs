#[fortuples::auto_impl]
trait Test {
    some_macro! {

    }
}

fn main() {}
