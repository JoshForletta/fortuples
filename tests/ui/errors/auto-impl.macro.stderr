error: `auto_impl` supports only methods
 --> tests/ui/errors/auto-impl.macro.rs:3:5
  |
3 |     some_macro! {
  |     ^^^^^^^^^^
